### PR TITLE
Ryochan7.DS4Windows: new package

### PR DESCRIPTION
**Pull request by soredake**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/126846)

### CODE REVIEW

**Comment by soredake** [comment]
Based on https://github.com/microsoft/winget-pkgs/pull/125780

File: manifests/r/Ryochan7/DS4Windows/3.2.19/Ryochan7.DS4Windows.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Ryochan7.DS4Windows
+PackageVersion: 3.2.19
+InstallerType: nullsoft
+Scope: machine
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+    - PackageIdentifier: Microsoft.DotNet.Runtime.6
```
Comment (Trenly) [inline manifests/r/Ryochan7/DS4Windows/3.2.19/Ryochan7.DS4Windows.installer.yaml:11]:
```suggestion
    - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.6
```

**Review by Trenly — COMMENTED** [review]
Seems you don't have the right dependencies; Also, dependencies are not yet enabled in the pipelines

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/r/Ryochan7/DS4Windows/3.2.19/Ryochan7.DS4Windows.installer.yaml (hunk 1 above)
Comment (soredake) [reply manifests/r/Ryochan7/DS4Windows/3.2.19/Ryochan7.DS4Windows.installer.yaml]: Updated, thanks!

**Review by stephengillie — APPROVED** [review]

**Pull request merged**